ts/modules/sankey.src.ts

```diff
@@ -648,7 +648,12 @@ seriesType<Highcharts.SankeySeries>(
                     totalNodeOffset;
 
                 for (var i = 0; i < column.length; i++) {
-                    totalNodeOffset = column[i].getSum() * factor + nodePadding;
+                    if (column[i].getSum()) {
+                        totalNodeOffset = column[i].getSum() * factor + nodePadding;
```

Comment:
Put the results of `column[i].getSum()` into a variable (or rather a constant) so that we don't have to compute it twice.